core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/SubmissionComponent.scala

```diff
@@ -279,8 +279,11 @@ trait SubmissionComponent {
         })
       )
 
-    def listAllActiveSubmissionIdsWithWorkspace(): ReadAction[Seq[(UUID, WorkspaceName)]] = {
-      val query = findActiveSubmissions join workspaceQuery on (_.workspaceId === _.id)
+    def listRecentActiveSubmissionIdsWithWorkspace(): ReadAction[Seq[(UUID, WorkspaceName)]] = {
+      // Exclude submissions from monitoring if they are ancient/stuck [WX-820]
+      // Empirically as of 3/23 there were no successful submissions that lasted longer than ~20 days
+      val cutoffTime = new Timestamp(DateTime.now().minusDays(90).getMillis)
```

Comment:
Do we want the cutoff here to be configurable? 90 days seems plenty conservative, so its probably not strictly necessary, but I could see this being set in a config so its a little easier to track in the future.

Reply:
Love the config idea, implementing now. I definitely meant to do 30 days and committed the wrong line.